app/code/Magento/Customer/Model/Backend/Search/Customer.php

```diff
@@ -3,32 +3,19 @@
  * Copyright © Magento, Inc. All rights reserved.
```

Comment:
Unfortunately a class can not be moved, since already existing 3rd-party code may already rely on it.